application/controllers/collection_instrument.py

```diff
@@ -136,24 +135,51 @@ def get_instruments_by_exercise_id_csv(exercise_id, session=None):
 
     @staticmethod
     @with_db_session
-    def get_instrument_json_by_id(instrument_id, session=None):
+    def get_instrument_json(instrument_id, session):
         """
-        Get collection instrument from the db using the id
+        Get collection instrument json from the db
         :param instrument_id: The id of the instrument we want
-        :param session: database session
         :return: formatted JSON version of the instrument
         """
 
-        log.info('Searching for instrument using id {}'.format(instrument_id))
+        instrument = CollectionInstrument.get_instrument_by_id(instrument_id, session)
+
         instrument_json = None
-        validate_uuid([instrument_id])
+        if instrument:
+            instrument_json = instrument.json
```

Comment:
These 3 line can be replaced with 
`instrument_json = instrument.json if instrument else None`

Reply:
Never a massive fan of the readability of this approach